work/v1/types.go

```diff
@@ -115,6 +139,14 @@ type ManifestWorkStatus struct {
 	// spoke cluster. The agent on spoke cluster syncs the condition from spoke to the hub.
 	// +optional
 	ResourceStatus ManifestResourceStatus `json:"resourceStatus,omitempty" protobuf:"bytes,2,rep,name=resourceStatus"`
+
+	// CreatedResources represents every resource that has been created for this ManifestWork and has not yet been deleted.
```

Comment:
will `CreatedResources` include the resource that  already  exist in the spoke cluster, so manifestwork is defined to update that resource rather than create it?

Reply:
> will `CreatedResources` include the resource that already exist in the spoke cluster, so manifestwork is defined to update that resource rather than create it?

Well considered question.  I think so, yes.  On a successful mutation call, the `CreatedResources` will be updated.  Can you think of a better name or a better way to phrase the doc?

Reply:
`TrackedResources` ?